cinema_service/settings.py

```diff
@@ -20,12 +22,11 @@
 # See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = (
-    "django-insecure-ru6ndfka@95_(lysua8yhdjq@vpiqgv3yru4r)q3h4_u8x7dfy"
-)
+
+SECRET_KEY = config("SECRET_KEY")
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = False
 
 ALLOWED_HOSTS = []
```

Comment:
The `ALLOWED_HOSTS` list is currently empty. For production, you should specify the domain names or IP addresses that are allowed to serve the application to prevent HTTP Host header attacks.